[docs] Unable to install datepicker

<!-- Provide a general summary of the issue in the Title above -->

I'm having problems installing datepicker (also the new one).

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/mui-org/material-ui/issues) of this repository and believe that this is not a duplicate.

## Current Behavior 😯

The guide at 'https://next.material-ui.com/guides/pickers-migration/' explains that al l that needs to be done is install the package @material-ui/lab. All required datepicker modules are unable to resolve:

```
  Line 9:28:   Unable to resolve path to module '@material-ui/lab/AdapterDateFns'        import/no-unresolved
  Line 10:34:  Unable to resolve path to module '@material-ui/lab/LocalizationProvider'  import/no-unresolved
  Line 11:24:  Unable to resolve path to module '@material-ui/lab/DatePicker'            import/no-unresolved
```

## Steps to Reproduce 🕹

```
  npm install @material-ui/core
  npm install @material-ui/labs

  import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
  import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
  import DatePicker from '@material-ui/lab/DatePicker';
```


## Your Environment 🌎


<details>
  <summary>`npx @material-ui/envinfo`</summary>
  
```
  System:
    OS: macOS 11.2.1
  Binaries:
    Node: 14.15.0 - /usr/local/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 7.5.4 - /usr/local/bin/npm
  Browsers:
    Chrome: 89.0.4389.82
    Edge: Not Found
    Firefox: 77.0.1
    Safari: 14.0.3
  npmPackages:
    @material-ui/core: ^4.11.3 => 4.11.3 
    @material-ui/icons: ^4.11.2 => 4.11.2 
    @material-ui/lab: ^4.0.0-alpha.57 => 4.0.0-alpha.57 
    @material-ui/styles:  4.11.3 
    @material-ui/system:  4.11.3 
    @material-ui/types:  5.1.0 
    @material-ui/utils:  4.11.2 
    @types/react: ^17.0.0 => 17.0.2 
    react: ^17.0.1 => 17.0.1 
    react-dom: ^17.0.1 => 17.0.1 
    typescript: ^4.1.2 => 4.2.3 
```
</details>

@eps1lon Installed that exact way same issue
```

  System:
    OS: Windows 10 10.0.19042
  Binaries:
    Node: 12.18.2 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.14.5 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 89.0.4389.82
    Edge: Spartan (44.19041.423.0), Chromium (89.0.774.54)
```
was on Material ui
```
"@material-ui/core": "^4.11.1",
    "@material-ui/icons": "^4.9.1",
    "@material-ui/lab": "^4.0.0-alpha.56",
```
Migrated to 
```
"@material-ui/core": "^5.0.0-alpha.27",
    "@material-ui/icons": "^5.0.0-alpha.27",
    "@material-ui/lab": "^5.0.0-alpha.27",
```
No webpack changes but webpack is now telling me that several files are not found. Looks like files using @material-ui

other errors 
```
 Module not found: Error: Can't resolve '@material-ui/lab/MobileDatePicker' in '/var/www/src/containers/Auth'
Module not found: Error: Can't resolve '@material-ui/lab/LocalizationProvider' in '/var/www/src/containers/Auth'
Module not found: Error: Can't resolve '@material-ui/lab/AdapterDateFns' in '/var/www/src/containers/Auth'
```

--- a/docs/src/pages/guides/pickers-migration/pickers-migration.md
+++ b/docs/src/pages/guides/pickers-migration/pickers-migration.md
@@ -8,7 +8,7 @@ This guide is an overview of the core concepts that were changed from pickers v3
 
 ## Installation
 
-You need to install the `@material-ui/lab` package if it's not already installed. Nothing more is required.  
+You need to install the `@material-ui/lab` package if it's not already installed.
 ⚠️ Make sure you have installed `"@material-ui/lab": ^5.0.0-alpha.16"` or above.
 
 ## Imports

--- a/docs/src/pages/guides/pickers-migration/pickers-migration.md
+++ b/docs/src/pages/guides/pickers-migration/pickers-migration.md
@@ -9,7 +9,7 @@ This guide is an overview of the core concepts that were changed from pickers v3
 ## Installation
 
 You need to install the `@material-ui/lab` package if it's not already installed.
-⚠️ Make sure you have installed `"@material-ui/lab": ^5.0.0-alpha.16"` or above.
+⚠️ Make sure you have installed the latest version, `"@material-ui/lab": ^5.0.0-alpha.30"` or above.
 
 ## Imports
 

--- a/docs/src/pages/guides/pickers-migration/pickers-migration.md
+++ b/docs/src/pages/guides/pickers-migration/pickers-migration.md
@@ -8,7 +8,8 @@ This guide is an overview of the core concepts that were changed from pickers v3
 
 ## Installation
 
-You simply need to install the `@material-ui/lab` package if it's not already installed. Nothing more is required.
+You need to install the `@material-ui/lab` package if it's not already installed. Nothing more is required.  
+⚠️ Make sure you have installed `"@material-ui/lab": ^5.0.0-alpha.16"` or above.
 
 ## Imports
 

